core/src/test/java/org/bitcoinj/script/ScriptChunkTest.java

```diff
@@ -46,4 +50,26 @@ public void testShortestPossibleDataPush() {
         assertFalse("push of 255 bytes", new ScriptChunk(OP_PUSHDATA2, new byte[255]).isShortestPossiblePushData());
         assertFalse("push of 65535 bytes", new ScriptChunk(OP_PUSHDATA4, new byte[65535]).isShortestPossiblePushData());
     }
+
+    @Test
+    @Ignore
+    public void testGetData() {
```

Comment:
I've evaluated the overhead of invoking `getData()` in this way. It seems that no overhead is introduced.